elements/rh-secondary-nav/demo/analytics.html

```diff
@@ -0,0 +1,209 @@
+<link rel="stylesheet" href="../demo.css">
+<link rel="stylesheet" href="../../rh-secondary-nav-lightdom.css">
+<script type="module" src="../rh-secondary-nav.js"></script>
+
+<div class="mock-nav"></div>
+<rh-secondary-nav role="navigation" data-analytics-region="Ansible secondary nav">
```

Comment:
@jvprh Can you make the region something like data-analytics-region="secondary-navigation". Do not want it to be site specific since we will use the same tracking across all.

Reply:
@rmessickSDI - The secondary navs will not be site specific, they will be site _section_ specific. There are currently at least 6 different iterations of this nav that will be replaced by this component. The best examples are those found on the RHEL, Ansible and Openshift product pages.  Does that change the recommendation?

Reply:
Oh, that makes sense.  Can we make it like this with the section being the last?  That way we can set one rule for all and have the option to get specific with section if we need to.  `data-analytics-region="secondary-navigation-ansible"`

Reply:
Yes, but now I have to ask about your use of dashes here vs no dashes in *-category

Reply:
Sure.  The category is data values that will be put directly into analytics.  The region is just used for logic in Launch.  That logic is text string based prefer no spaces.  So for this instance just like the homepage sections, the hierarchy is the other way to make it easy for regex functions.
example:
secondary-navigation-ansible
secondary-navigation-Openshift
secondary-navigation-RHEL
I can make 1 launch condition to start with "secondary-navigation-" and have event fire across all of them.  Then also have the option to fire the specific events if business needs all the way down to section level.

Did the same with redhat.com home page sections:
homepage-offer-card-featured

Reply:
Got it. 👍

Reply:
updated!

Reply:
Having received @rmessickSDI's approval via email, resolving this and merging